### DISCUSSION TITLE
Unable to see the logs in airflow UI

### DISCUSSION THREAD

**Discussion by vedprakash0505** [apache/airflow#25125] [Q&A]
After upgrading airflow 2.1.1 to 2.2.3, we started seeing issue while seeing the tasks logs on UI.

we have tried with the config to set "web server secret key" in values.yml

webserverSecretKey: "134b0d88-041e-11"

*** Log file does not exist: /opt/airflow/logs/lovenish_sample2/mid/2022-07-15T09:31:29.163998+00:00/1.log *** Fetching from: http://airflow-worker-0.airflow-worker.airflow.svc.cluster.local:8793/log/lovenish_sample2/mid/2022-07-15T09:31:29.163998+00:00/1.log *** !!!! Please make sure that all your Airflow components (e.g. schedulers, webservers and workers) have the same 'secret_key' configured in 'webserver' section !!!!! ****** See more at https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#secret-key ****** Failed to fetch log file from worker. 403 Client Error: FORBIDDEN for url: http://airflow-worker-0.airflow-worker.airflow.svc.cluster.local:8793/log/lovenish_sample2/mid/2022-07-15T09:31:29.163998+00:00/1.log For more information check: https://httpstatuses.com/403

**Comment by potiuk** [comment]
Did you follow the advices/links there? What did you find?

**Reply by vedprakash0505** [reply]
As per https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#secret-key  , we checked for time on all machine and it was same for all . we tried to set webserverSecretKey in [values.yaml](https://github.com/apache/airflow/blob/main/chart/values.yaml) with 16 digit unicode string but that also does not work.

**Reply by vedprakash0505** [reply]
it worked after setting following env in values.yaml
env:
  - name: "AIRFLOW__WEBSERVER__SECRET_KEY"
    value: "<16 digit unicode>"

**Reply by potiuk** [reply]
If you are using the Official Helm Chart you should set webserverSecretKey instead.

Setting the variable is generally a bad idea for secret because it will be plain-open in your deployment.

The `webserverSecretKey` should work after redeploying the chart. It should generate a Kubernetes secret that should be used by both webserver and workers. Are you sure you followed https://airflow.apache.org/docs/helm-chart/stable/production-guide.html#webserver-secret-key to set it properly (it shoudl be generated using the provided instructions).

What exactly did not work when you tried to set it? Did you get any error messages (badly formatted key? or smth?)
